rfcs/text/0000-planning-for-the-dcp.md

```diff
@@ -0,0 +1,221 @@
+
+***NOTE**: In the future, DCP roadmaps will be a source for a shared understanding of DCP priorities.*
+
+After this meeting, the **Product Owner** and **Technical Lead** for each component MUST ensure that the *Product Backlog* in their repo is refined and aligned to the DCP priorities.
+
+(See *Refinement* in [*Scrum Guide*](https://scrumguides.org/docs/scrumguide/v2017/2017-Scrum-Guide-US.pdf) and [*Nexus Guide*](http://scrumorg-website-prod.s3.amazonaws.com/drupal/2018-01/2018-Nexus-Guide-English_0.pdf) for guidance.)
+
+#### Refinement and ZenHub Pipelines
+
+New items in the *dcp* repo appear in the ZenHub *New* pipeline and MUST be triaged into the *Product Backlog* pipeline, the *Icebox* pipeline, or the *Closed* pipeline.
+
+The ZenHub *Product Backlog* pipeline is the top-level DCP product backlog which is ordered during Refinement.
+
+The ZenHub *Icebox* pipeline SHOULD be used with great constraint in the *dcp* repo. It's not a pipeline for avoiding hard decisions during Refinement. It's intended to document potential *futures* from the DCP roadmap. Individual component repo(s) MAY use the *Icebox* pipeline in a manner of their own choosing.
+
+No issues in the *dcp* repo MUST be assigned to the *Epic* pipeline. Individual component repo(s) MAY use the *Epic* pipeline in a manner of their own choosing.
+
+### Sprint Planning
+
+Prior to the start of the DCP Sprint, component teams MUST identify their *Sprint Backlog* - a slice of items from the the ZenHub *Product Backlog* pipeline in their repo which are assigned to the ZenHub Milestone for the Sprint and moved to the ZenHub *Sprint Backlog* pipeline.
+
+The number of items selected is solely up to the Development Team. 
+- [ZenHub story points](https://help.zenhub.com/support/solutions/articles/43000034282) MUST be assigned to all *Sprint Backlog* items. (Story points are required to enable [ZenHub burndown charts](https://help.zenhub.com/support/solutions/articles/43000010356-track-sprint-progress-with-burndown-charts) to track the progress of a Sprint.)
+  - [How to start?](https://www.scrum.org/forum/scrum-forum/6774/how-decide-how-many-story-points-take-first-sprint)
+- Research and prototyping tasks SHOULD be modeled as Spikes.
+- Dependencies between components within a Sprint MUST be communicated between the respective teams and surfaced using [ZenHub Dependencies](https://help.zenhub.com/support/solutions/articles/43000010349). 
+
+At the start of the Sprint, the single DCP Sprint Backlog SHOULD be reviewed by the **PM Team** and **Architecture Team** in their oversight roles.
+
+### Sprint(ing)
+
+During Development, the Development Team assigns issues from the ZenHub *Sprint Backlog* pipeline and move to the *In Progress* pipeline. 
```

Comment:
The use of "Development" here …

Reply:
Easy change - _During Development_ -> _During the Sprint_ to avoid the collision below.

Reply:
There are two occurrences of that word in the sentence.